notebooks/development/Build Breaker identification - PIP.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>This is ok for now, but we should consider introduce a new package that would encapsulate this build breaker analysis and place all there relevant logic there - including the log parser you used which is in user-api.</p>

 _Reply via <a href='https://app.reviewnb.com/thoth-station/notebooks/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Definitely, I'd like to put it into thoth-lab sub-package which would encapsulate utilities like this one.</p>

Reply:
A standalone library would be more appropriate.